doc/pyang.1.dbk

```diff
@@ -937,6 +971,108 @@
     </variablelist>
   </refsect1>
 
+  <refsect1 xml:id="man.1.pyang.sid_options">
+    <title>YANG Schema Item iDentifiers (SID)</title>
+    <para>
+      YANG Schema Item iDentifiers (SID) are globally unique unsigned integers
+      used to identify YANG items. SIDs are used instead of names to save space
+      in constrained applications such as COREconf. This plugin is used to
+      automatically generate and updated .sid files used to persist and
+      distribute SID assignments.
+    </para>
+    <para>
+      Options for generating, updating and checking .sid files:
+    </para>
+    <variablelist>
+      <varlistentry>
+        <term><option>--generate-sid-file</option></term>
+        <listitem>
+          <para>
+            This option is used to generate a new .sid file from a YANG module. This example
+            shows how to generate the file <emphasis>toaster@2009-11-20.sid</emphasis>.
+          </para>
+          <informalexample>
+            <screen>pyang --generate-sid-file 20000:100 toaster@2009-11-20.yang</screen>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--update-sid-file</option></term>
```

Comment:
document the argument in the <option> i.e.  <replaceable>prevsidfile<replaceable>